modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -253,10 +265,40 @@ struct IEUnit {
                 }
             }
         }
-#endif
-
+#endif // INF_ENGINE_RELEASE < 2020000000
+        // In dnn, after ordinary extensions they try to add custom <InfEngineExtension> as
+        // "some of networks can work without a library of extra layers.
+        //  OpenCV fallbacks as extensions"
+        // Do WE need this? (dnn/src/op_inf_engine.cpp:824)
+#if INF_ENGINE_RELEASE < 2019020000  // < 2019.R2
         auto this_network = this_plugin.LoadNetwork(net, {}); // FIXME: 2nd parameter to be
                                                               // configurable via the API
+#else
+        bool isHetero = false;
+        if (params.device_id != "CPU")
+        {
+            isHetero = params.device_id == "FPGA";
+            for (auto& layer : net)
```

Comment:
does it really work with latest master? It requires `begin()` / `end()` methods for `CNNNetwork` which are deprecated and removed on current master. 
Secondly, we don't we work with ngraph directly? If we speak about IR v10 which is default since 2020.1

Reply:
fixed